main.tf

```diff
@@ -477,3 +477,56 @@ resource "aws_cloudwatch_log_stream" "cloudwatch_stream" {
   name           = "${var.app_name}-log-stream"
   log_group_name = aws_cloudwatch_log_group.cloudwatch_group.name
 }
+
+resource "aws_autoscaling_schedule" "ecs_non_prod_scale_down" {
+  count                  = local.is-production == true ? 1 : 0
+  scheduled_action_name  = "ecs_non_prod_scale_down"
+  min_size               = 0
+  max_size               = 0
+  desired_capacity       = 0
+  recurrence             = "0 20 * * *" # 20.00 UTC time or 21.00 London time
+  autoscaling_group_name = aws_autoscaling_group.ecs_non_prod_daily.name
+}
+
+resource "aws_autoscaling_schedule" "ecs_non_prod_scale_up" {
+  count                  = local.is-production == true ? 1 : 0
+  scheduled_action_name  = "ecs_non_prod_scale_up"
+  min_size               = 1
+  max_size               = 1
```

Comment:
this must use user defined  min/max_size, and should not equal to 1 by default (customers might have different needs for their applications)